src/interface/window.c

```diff
@@ -1354,7 +1354,7 @@ void sub_688956()
  * 
  *  rct2: 0x0068881A
  */
-void window_rotate_camera(rct_window *w)
+void window_rotate_camera(rct_window *w, int wise)
```

Comment:
i think `direction` would be a better name for this. And add a little comment saying what `direction` is used for.